packages/gestalt/src/SlimBanner.js

```diff
@@ -0,0 +1,100 @@
+// @flow strict
+import { type Node } from 'react';
+import Box from './Box.js';
+import Icon from './Icon.js';
+import Link from './Link.js';
+import Text from './Text.js';
+import Flex from './Flex.js';
+import { MESSAGING_TYPE_ATTRIBUTES } from './messaging.js';
+
+type Props = {|
+  /**
+   * Helper [Link](https://gestalt.pinterest.systems/link) to be placed after the message. See the [helperLink variant](https://gestalt.pinterest.systems/slimbanner#helperLink) to learn more.
+   */
+  helperLink?: {|
+    text: string,
+    accessibilityLabel: string,
+    href: string,
+    onClick: ({|
+      event: SyntheticMouseEvent<HTMLAnchorElement> | SyntheticKeyboardEvent<HTMLAnchorElement>,
+      dangerouslyDisableOnNavigation: () => void,
+    |}) => void,
+  |},
+
+  /**
+   * Label to describe the status icon’s purpose. See the [Accessibility guidelines](https://gestalt.pinterest.systems/slimbanner#Accessibility) for details on proper usage.
+   */
+  iconAccessibilityLabel?: string,
```

Comment:
Could we use our provider to add this by default? I don't think they'd ever need to be anything more than "Warning", "Error", etc.

Reply:
aka, it would be great if we didn't need this prop at all

Reply:
+1

Reply:
refactored to combined props

Reply:
I'll refgacor in followup PR. 'll handle this in several components